### ISSUE TITLE
Funding

### ISSUE THREAD

**Issue by vsushkov** [olegkoval/magento2-regenerate_url_rewrites#116]
Could you add your PayPal details to the project, please? I'm not a big fan of subscriptions and therefore I don't like the Patreon. So I think it'd be awesome if you'll [add the "sponsor" button to your repository](https://help.github.com/en/github/building-a-strong-community/displaying-a-sponsor-button-in-your-repository).

**Comment by olegkoval** [comment]
Hello @vsushkov.

Thank you for this issue ticket. Problem is that I'm a resident of Montenegro and I can use a PayPal only to pay. I can't receive money. I think Russia had same usage rules before (now you can receive money). And Patreon is one of the platform which I can use to get a donations without a residence restrictions.

Best regards,
Oleg

**Comment by codedge** [comment]
Whats with Github Sponsor, would that work or is there also a residence restriction?

**Comment by olegkoval** [comment]
@codedge Yes, sponsorship via Github is not available for Montenegro residents. This is the list of allowed countries (only 30 countries):
https://help.github.com/en/github/supporting-the-open-source-community-with-github-sponsors/setting-up-github-sponsors-for-your-user-account#submitting-your-bank-and-tax-information 

If your country is not in EU or it's not a USA/Canada/etc. then it's hard to receive a money online via PayPal or donation systems which use Stripe/etc. billing systems, because your country just is not in the list.

Only way to get they money: bank transfer or something like Patreon (or be invited as a freelancer via Upwork ;) https://www.upwork.com/o/profiles/users/~01a40656da65c38d3a/ )

**Comment by olegkoval** [comment]
Hello @mehdichaouch .

Yes, I missed this.

Thank you and I've updated a README with BTC wallets info. Also I'll add this info into the next release.

BTC: bc1qssffnksrcfwalmg06n6dlr98vh9576hcer44q2
ETC: 0x297dE8348E4B8f6Ce7fF25D23bD8a8E60b26b969
BNB: bnb1r8gqxca0fsa2cyqr2e2g5eu4h3cdar57zp48xa

**Comment by olegkoval** [comment]
Support links (PayPal/Patreon/ko-fi) are already included in the README and shown in the CLI output (Helper\Regenerate::getSupportMeText()). Closing this discussion thread.
